code/_onclick/hud/radial.dm

```diff
@@ -309,4 +309,45 @@ GLOBAL_LIST_EMPTY(radial_menus)
 	GLOB.radial_menus -= uniqueid
 	return answer
 
+/**
+ * Similar to show_radial_menu, but choices is a list of atoms, for which icons will be automatically generated.
+ * Supports multiple items of the same name, 2 soaps will become soap (1) and soap (2) to the user
+ */
+/proc/radial_menu_helper(mob/user, atom/anchor, list/choices, uniqueid, radius, datum/callback/custom_check, require_near = FALSE)
```

Comment:
Can you document the arguments for this proc in the standard format? Would help other people use this down the line

Reply:
All of the arguments here are the same as for show_radial_menu except choices. I'm not exactly sure what all of them do in context to radial menus, and the original proc doesnt have context for them either.

Reply:
Fair enough